lib/python/treadmill_aws/bootstrap/node/aws.linux/bin/parts/99_remove_cgroups.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+AWK="{{ _alias.awk }}"
+CAT="{{ _alias.cat }}"
+ECHO="{{ _alias.echo }}"
+GREP="{{ _alias.grep }}"
+
+CGROUP_BASE="/sys/fs/cgroup"
+TREADMILL_ROOT_CGROUP=treadmill
+
+# get total physical memory in Bytes
+PHYSICAL_MEMORY_KB="$(${GREP} MemTotal /proc/meminfo | ${AWK} '{print $2}')"
+PHYSICAL_MEMORY="$((1024*$PHYSICAL_MEMORY_KB))"
+
+# Get total number of CPU cores
+CPUSET_ALL_CORES="$(${CAT} ${CGROUP_BASE}/cpuset/cpuset.cpus)"
+
+# Reset CPU limits
+${ECHO} 100 >${CGROUP_BASE}/cpu/system.slice/cpu.shares
+${ECHO} 100 >${CGROUP_BASE}/cpu/${TREADMILL_ROOT_CGROUP}/cpu.shares
+
+# Reset CPU core assignment
+${ECHO} ${CPUSET_ALL_CORES} >${CGROUP_BASE}/cpuset/system.slice/cpuset.cpus
+${ECHO} ${CPUSET_ALL_CORES} >${CGROUP_BASE}/cpuset/${TREADMILL_ROOT_CGROUP}/cpuset.cpus
+
+# Reset memory limits
+${ECHO} ${PHYSICAL_MEMORY} >${CGROUP_BASE}/memory/system.slice/memory.limit_in_bytes
+${ECHO} ${PHYSICAL_MEMORY} >${CGROUP_BASE}/memory/${TREADMILL_ROOT_CGROUP}/memory.limit_in_bytes
```

Comment:
we should not go oom... are we? i think this is extra and not needed.

Reply:
This was the original patch we put in place to prevent the OOM from occurring under the grid workload:
https://github.com/bretttegartms/treadmill/commit/0bc959197c6833e592b8bbf7a038ee16eeb8f689#diff-b140a318aff990ce62a490c80f568bb3

Reply:
well, i suspect this is related to memory miscalculation. given that we shared 2G between system and treadmill, with memory pressure from grid, it is not surprising that we oom.

...core]$ more memory.usage_in_bytes
734793728
system.slice]$ more memory.usage_in_bytes
223076352

this is without memory optimizations that are not committed...